experimental/torch_xla2/test/test_ops.py

```diff
@@ -303,6 +299,7 @@
   'uniform',
   # Dropout is not deterministic https://pytorch.org/docs/stable/generated/torch.nn.functional.feature_alpha_dropout.html
   'nn.functional.feature_alpha_dropout',
+  'histc', # result is not stable for single negative scalar
```

Comment:
try to remove it? so there is some randomness, in my lastest PR I added torch.manual_seed(0) and it seems to pass after that

Reply:
thanks, updated